src/main/java/Ui.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * A class to handle User Interactions.
+ */
+
+import java.util.Scanner;
+
+public class Ui {
+
+    /**
+     * A method that prints out the introduction when Duke first initialises.
+     */
+
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke \nWhat can I do for you?");
+    }
+
+    public String readCommand() {
+        Scanner sc = new Scanner(System.in);
```

Comment:
It may not be necessary to create a new Scanner object everytime readCommand() is called.